app/ide-desktop/lib/dashboard/src/components/AriaComponents/Text/useVisualTooltip.tsx

```diff
@@ -64,31 +88,40 @@ export function useVisualTooltip(props: VisualTooltipProps) {
   })
 
   const { overlayProps } = aria.useOverlayPosition({
+    isOpen: isHovered,
     overlayRef: popoverRef,
     targetRef,
     offset,
-    isOpen: isHovered,
     placement,
     containerPadding,
   })
 
   return {
     targetProps: aria.mergeProps<React.HTMLAttributes<HTMLElement>>()(hoverProps, { id }),
     tooltip: isDisabled ? null : (
-      <span
-        id={id}
-        ref={popoverRef}
-        className={ariaComponents.TOOLTIP_STYLES({
-          className: clsx(className, 'hidden animate-in fade-in [&:popover-open]:flex'),
-        })}
-        // @ts-expect-error popover attribute does not exist on React.HTMLAttributes yet
-        popover=""
-        aria-hidden="true"
-        role="presentation"
-        {...overlayProps}
-      >
-        {children}
-      </span>
+      <Portal>
+        <span
+          id={id}
+          ref={popoverRef}
+          className={ariaComponents.TOOLTIP_STYLES({
+            className: clsx(className, 'hidden animate-in fade-in [&:popover-open]:flex'),
```

Comment:
might as well replace `clsx` with `tailwind-merge`

Reply:
tw-merge is called under the hood